### DISCUSSION TITLE
2.0.0-20079 Release

### DISCUSSION THREAD

**Discussion by IacobIonut01** [IacobIonut01/Gallery#227] [Announcements]
## What's Changed
* Update agp to v8.2.0-rc02 by @renovate in https://github.com/IacobIonut01/Gallery/pull/216
* Update coilVersion to v2.5.0 by @renovate in https://github.com/IacobIonut01/Gallery/pull/217
* Imported new Translations from Crowdin
* Monthly timeline should be fixed
* Added fast scroll for timeline

**Full Changelog**: https://github.com/IacobIonut01/Gallery/compare/2.0.0-20070...2.0.0-20079

<hr /><em>This discussion was created from the release <a href='https://github.com/IacobIonut01/Gallery/releases/tag/2.0.0-20079'>2.0.0-20079 Release</a>.</em>

**Comment by fallsdevil** [comment]
Hey, amazing!

now, about the fancy blur option in android 11 is not selectable, is this a bug or a limitation of the android version?
